File: src/b1012.java
import java.util.Locale;
import java.util.Scanner;

public class b1012 {
    public static void main(String[] args) throws Exception {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double A, B, C, triangulo, circulo, trapezio, quadrado, retangulo;

        A = sc.nextDouble();
        B = sc.nextDouble();
        C = sc.nextDouble();

        triangulo = (A * C) / 2;
        circulo = 3.14159 * (Math.pow(C, 2));
        trapezio = ((A + B) * C) / 2;
        quadrado = Math.pow(B, 2);
        retangulo = A * B;

        System.out.println("TRIANGULO: " + String.format("%.3f", triangulo));
        System.out.println("CIRCULO: " + String.format("%.3f", circulo));
        System.out.println("TRAPEZIO: " + String.format("%.3f", trapezio));
        System.out.println("QUADRADO: " + String.format("%.3f", quadrado));
        System.out.println("RETANGULO: " + String.format("%.3f", retangulo));

        sc.close();
    }
}

File: Iniciantes/b1012.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

const pi = 3.14159;

var line1 = lines.shift().split(' ');
var A = parseFloat(line1[0]);
var B = parseFloat(line1[1]);
var C = parseFloat(line1[2]);

var triangulo = (A * C) / 2;
var circulo = pi * Math.pow(C, 2);
var trapezio = ((A + B) * C) / 2;
var quadrado = Math.pow(B, 2);
var retangulo = A * B;

console.log('TRIANGULO: ', triangulo.toFixed(3));
console.log('CIRCULO: ', circulo.toFixed(3));
console.log('TRAPEZIO: ', trapezio.toFixed(3));
console.log('QUADRADO: ', quadrado.toFixed(3));
console.log('RETANGULO: ', retangulo.toFixed(3));
